redirects.json

```diff
@@ -843,5 +843,35 @@
     "source": "/advanced/passkeys/supported-networks",
     "destination": "/advanced/smart-account-supported-networks?module=Safe+Passkey+Module",
     "permanent": true
+  },
```

Comment:
As all the new "sources" listed below are not pages, these redirects shouldn't be needed, no?

Reply:
I believe all URL segments (like folders) should either have an index page or be redirected to the first page of the folder. It is very likely that users will try to navigate to these pages (or links wrongfully point to the folder instead of an actual page).

Reply:
I agree with Germán here for the reasons he mentioned in his last comment.